src/modules/dapp-staking/utils/index.ts

```diff
@@ -1,4 +1,60 @@
+import BN from 'bn.js';
+import { StakeInfo, EraStakingPoints } from './../../../store/dapp-staking/actions';
 import { ApiPromise } from '@polkadot/api';
+import { DappItem } from './../../../store/dapp-staking/state';
+import { GeneralStakerInfo } from 'src/hooks/helper/claim';
+import { ContractEvm, StakingData } from './../index';
+import { balanceFormatter } from 'src/hooks/helper/plasmUtils';
+import { EraIndex } from '@polkadot/types/interfaces';
+import { Option } from '@polkadot/types';
+
+const getDapps = async (api: ApiPromise): Promise<string[]> => {
+  try {
+    const dApps = (await api.query.dappsStaking.registeredDapps.entries()).map((it) => {
+      const dapp = it[0].toHuman() as ContractEvm[];
```

Comment:
Try to avoid toHuman if possible

Reply:
Fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/385/commits/bb9a912dab72519f55f7650f6f23f03c638829ef) commit.